tests/integration/test_yarn_classic.py

```diff
@@ -113,7 +113,7 @@ def test_yarn_classic_packages(
         pytest.param(
             utils.TestParameters(
                 repo="https://github.com/cachito-testing/cachi2-yarn.git",
-                ref="67679cb740122cfa6c17238c7a1ff9cfcdb875f3",
+                ref="4fe0919c4b2fb9a0f6fb98999ceaeb732f4b1d9d",
```

Comment:
Do we prefer that? I don't have a strong opinion, but I did like the updates to be more visible/deliberate

Reply:
Every change to a test requires a corresponding change in the itegration suite runner.

> Do we prefer that?

We recently did and the last several PRs that we merged that included integration test changes already use branch names.

Reply:
> Do we prefer that?

I would strongly recommend to in case we don't. It is so much easier to maintain tests with human-friendly refs than with hashes.

Reply:
Sorry, github collapsed the comment and I forgot about it. Fixed